### PR TITLE
Improving introductory information in README

### PR DESCRIPTION
**Pull request by merltron-pa**
- I improved the general description of Herbie to be more accessible to a wider audience.

- The quick start in the README wasn't really a quick start, since you couldn't actually log in to the admin console, and the URL was wrong anyway.

- I update the URL and added steps that show you how to create your first admin user so that you can actually log in.

I basically edited everything up to the procedure "Run Herbie on Docker".

(The procedure "Run Herbie on Docker" and all the content that follows it could still be improved, but I will do that in a follow-up round.)

I also added fixes suggested by Lars here: https://github.com/project-a/herbie/pull/128

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,97 +2,160 @@
...
-    ├── business_entity1
-    │   ├── business_entity1_v1.json
-    │   └── business_entity1_v2.json    
-    └── business_entity2
+---business_entities_schemas
+        init.py 
+        ------  business_entity1
+                    ------- business_entity1_v1.json
+                    ------- business_entity1_v2.json    
+        ------  business_entity2
```
Comment (tobi812) [inline README.md]: I updated the presentation of the directory structure here. So maybe you can remove this update.

**Review by tobi812 — CHANGES_REQUESTED** [review]
Just a small thing besides that it looks good to me :+1:

**Review by tobi812 — APPROVED** [review]
Looks good to me :+1:  Thanks a lot!

File: README.md (hunk 1 above)
Comment (merltron-pa) [reply README.md]: @tobi812 OK, added in your changes

**Pull request merged**